Support for Eleventy 1.0

I just tried upgrading our existing Eleventy project to [Eleventy 1.0 Beta](https://www.11ty.dev/blog/eleventy-v1-beta/) and ran into some errors due to eleventy-plugin-i18n [currently requiring Eleventy prior to v1.0.0](https://github.com/adamduncan/eleventy-plugin-i18n/blob/3edce1a7121353d4aafaee73042db514851f5e61/package.json#L26):

```
$ npm install @11ty/eleventy@beta --save-dev
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eleventy-plugin-i18n@0.1.2
npm ERR! Found: @11ty/eleventy@1.0.0-beta.1
npm ERR! node_modules/@11ty/eleventy
npm ERR!   dev @11ty/eleventy@"1.0.0-beta.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @11ty/eleventy@">=0.11.0 <1.0.0" from eleventy-plugin-i18n@0.1.2
npm ERR! node_modules/eleventy-plugin-i18n
npm ERR!   dev eleventy-plugin-i18n@"^0.1.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @11ty/eleventy@0.12.1
npm ERR! node_modules/@11ty/eleventy
npm ERR!   peer @11ty/eleventy@">=0.11.0 <1.0.0" from eleventy-plugin-i18n@0.1.2
npm ERR!   node_modules/eleventy-plugin-i18n
npm ERR!     dev eleventy-plugin-i18n@"^0.1.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/.eleventy.js
+++ b/.eleventy.js
@@ -1,4 +1,4 @@
-const pkg = require("./package.json");
+const pkg = require('./package.json');
 const i18n = require('./i18n.js');
 
 const defaultOptions = {
@@ -8,9 +8,11 @@ const defaultOptions = {
 
 module.exports = function (eleventyConfig, configGlobalOptions = {}) {
   try {
-    eleventyConfig.versionCheck(pkg["11ty"].compatibility);
+    eleventyConfig.versionCheck(pkg['11ty'].compatibility);
   } catch (e) {
-    console.log(`WARN: Eleventy Plugin (${pkg.name}) Compatibility: ${e.message}`);
+    console.log(
+      `WARN: Eleventy Plugin (${pkg.name}) Compatibility: ${e.message}`
+    );
   }
   const pluginOptions = Object.assign({}, defaultOptions, configGlobalOptions);
 

--- a/.eleventy.js
+++ b/.eleventy.js
@@ -1,3 +1,4 @@
+const pkg = require("./package.json");
 const i18n = require('./i18n.js');
 
 const defaultOptions = {
@@ -6,6 +7,11 @@ const defaultOptions = {
 };
 
 module.exports = function (eleventyConfig, configGlobalOptions = {}) {
+  try {
+    eleventyConfig.versionCheck(pkg["11ty"].compatibility);
+  } catch (e) {
+    console.log(`WARN: Eleventy Plugin (${pkg.name}) Compatibility: ${e.message}`);
+  }
   const pluginOptions = Object.assign({}, defaultOptions, configGlobalOptions);
 
   eleventyConfig.addFilter('i18n', function (key, data, localeOverride) {

--- a/i18n.js
+++ b/i18n.js
@@ -11,10 +11,8 @@ module.exports = function (
   pluginOptions = {},
   page
 ) {
-  const {
-    translations = {},
-    fallbackLocales: fallbackLocales = {}
-  } = pluginOptions;
+  const { translations = {}, fallbackLocales: fallbackLocales = {} } =
+    pluginOptions;
 
   // Use explicit `locale` argument if passed in, otherwise infer it from URL prefix segment
   const url = get(page, 'url', '');

--- a/package.json
+++ b/package.json
@@ -22,8 +22,8 @@
     "url": "https://github.com/adamduncan/eleventy-plugin-i18n/issues"
   },
   "homepage": "https://github.com/adamduncan/eleventy-plugin-i18n#readme",
-  "peerDependencies": {
-    "@11ty/eleventy": "^1.0.0-beta.1"
+  "11ty": {
+    "compatibility": ">=0.11"
   },
   "dependencies": {
     "chalk": "^4.1.1",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "eleventy-plugin-i18n",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "Eleventy plugin to assist with dictionary translations",
   "main": ".eleventy.js",
   "scripts": {
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/adamduncan/eleventy-plugin-i18n#readme",
   "peerDependencies": {
-    "@11ty/eleventy": ">=0.11.0 <1.0.0"
+    "@11ty/eleventy": "^1.0.0-beta.1"
   },
   "dependencies": {
     "chalk": "^4.1.1",

